checks/runtimeCheck.go

```diff
@@ -0,0 +1,90 @@
+package checks
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/newrelic/newrelic-lambda-extension/util"
+)
+
+var (
+	layerAgentPathNode    = []string{"/opt/nodejs/node_modules/newrelic"}
+	layerAgentPathsPython = []string{
+		"/opt/python/lib/python2.7/site-packages/newrelic",
+		"/opt/python/lib/python3.6/site-packages/newrelic",
+		"/opt/python/lib/python3.7/newrelic",
+		"/opt/python/lib/python3.8/site-packages/newrelic",
+	}
```

Comment:
Are we committing ourselves to extension updates every time there's a new python rev? Can we avoid that?

Reply:
Path would need to be a glob. A Python runtime arrives maybe every 1.5-2 years, seemed like a decent tradeoff. Also, we would need to start publishing layers for a new Python runtime to support it, so changes downstream would happen anyway.

Not opposed to a glob approach. Just more expensive and added complexity.